### DISCUSSION TITLE
Asia Pacific friendly meeting time

### DISCUSSION THREAD

**Discussion by ralf0131** [open-telemetry/opentelemetry-java-instrumentation#11272] [Q&A]
Hi All, I am an engineer from Alibaba Cloud who is working on observability. I noticed that the Java SIG biweekly meeting is held at every Thursday at 09:00 PT, which is 0:00 Friday at midnight in Beijing time. This make us difficult to participant.
Is it possible to have an Asia-Pacific friendly meeting there? We are interested in contributing to the Open Telemetry project, and are looking forward to have more discussion with the community.

**Comment by steverao** [comment]
I also have the same problem!

**Comment by trask** [comment]
I'm up for scheduling a second SIG meeting that is APAC friendly. Maybe we can meet every other week Friday 8am China Standard Time (Thursday 5pm US Pacific Time)?

**Reply by ralf0131** [reply]
Hi, glad to hear that, it looks good to me :)

**Comment by codefromthecrypt** [comment]
8am is usually a good time, as singapore (also perth) are same timezone. In fact, this timezone (UTC +8) is the most populous on the planet.

I hope that we can lean into it not just for the folks like me speaking up, but also diversifying the leads in the group, across all SIGS. Everyone should be able to have a healthy sleep pattern, and have a chance to participate actively. While some SIGs may be small, the larger and established ones ideally are in a position that supporting APAC doesn't mean someone in new york having a really long day. In other words, ideally folks can split responsibilities such that everyone can sleep or not have a really hard technical discussion and try to immediately sleep after it.

TL;DR; even if my work isn't mostly focused on the java side, I would attend this just to thank folks for doing it, and hope that it doesn't get rolled back like the collector one did to 11pm again.

(end soapbox)

**Reply by codefromthecrypt** [reply]
ps I think this is a *very* important topic to discuss even if informally at KubeCon China.

I had a chat on the collector slack and believe that we are better off making this UTC+8 compelling, so that people will want to use the time to discuss primary topics. Otherwise, it becomes a review session and less valuable and certainly secondary.

Some ideas are to find champions in west coast US canada or latin america who are willing to share their impact by presenting topics exclusively in this meeting. 4pm/8am is very doable both sides. What I learned from the collector slack was that the asia time meeting wasn't well attended, and that topics likely wouldn't have been able to be held primarily in that timezone due to lack of maintainers. Basically, even if it is conjecture, I feel that if we brought some folks in who don't live in APAC, but are timezone friendly to it, we could establish enduring value, and at the same time grow contributors to the point where this side of the world has enough to close topics.

**Reply by trask** [reply]
Great to hear more interest in this! @steverao @ralf0131 @jack-berg and I just discussed this yesterday, and we are planning to launch a biweekly Java SIG meeting on Wednesday 6pm US Pacific / Thursday 9am China Standard Time starting the week after next (July 10/11).

**Reply by codefromthecrypt** [reply]
count me in. the collector meeting might be same day, an hour prior which is fine for me. still pending an otel champion to commit to running the collector one, which would confirm the date/time. 🤞

**Reply by codefromthecrypt** [reply]
ps cross-linking to the collector thing. It feels like this problem is very cross-cutting as rationale for things like this are not SIG specific https://github.com/open-telemetry/opentelemetry-collector/issues/7085#issuecomment-2192948097

@svrnm any ideas on how to sort of capture rationale and approach and solve once for SIGs large enough to span multiple regions sustainably?

**Reply by svrnm** [reply]
This is a topic we should look into with a [Contributor Experience SIG](https://github.com/open-telemetry/community/pull/2162) but even before creating it, this is something worth discussing via an issue on the community forum and share it via the #otel-maintainers slack + the Maintainer Meeting.

**Reply by codefromthecrypt** [reply]
@svrnm thanks, do you mean adding a discussion here summarizing this and the collector issues and suggested approach? https://github.com/open-telemetry/community/discussions

**Reply by svrnm** [reply]
I wanted to say to discuss it via an issue at https://github.com/open-telemetry/community/issues -- this is not a discussion, this is an issue that needs to be solved.

**Reply by codefromthecrypt** [reply]
as requested! https://github.com/open-telemetry/community/issues/2176

**Comment by trask** [comment]
see https://github.com/open-telemetry/community/pull/2197

first meeting is this Thu 9am China Standard Time, see you then!
